.github/workflows/e2e-compatibility.yaml

```diff
@@ -47,42 +48,29 @@ jobs:
           docker build . -t "${REGISTRY}/${ORG}/${IMAGE_NAME}:${{ needs.determine-docker-tag.outputs.docker-tag }}"
           docker push "${REGISTRY}/${ORG}/${IMAGE_NAME}:${{ needs.determine-docker-tag.outputs.docker-tag }}"
 
-  load-test-matrix:
-    outputs:
-      test-matrix: ${{ steps.set-test-matrix.outputs.test-matrix }}
-    needs: determine-docker-tag
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - run: |
-          # use jq -c to put the full json contents on a single line. This is required when using the json body
-          # to create the matrix in the following job.
-          test_matrix="$(cat .github/compatibility-test-matrices/${{ needs.determine-docker-tag.outputs.docker-tag }}.json | jq -c)"
-          echo $test_matrix
-          echo "::set-output name=test-matrix::$test_matrix"
-        id: set-test-matrix
+  transfer:
+    needs:
+      - build-release-image
+      - determine-docker-tag
+    uses: ./.github/workflows/e2e-compatibility-workflow-call.yaml
+    with:
+      docker-tag: "${{ needs.determine-docker-tag.outputs.docker-tag }}"
+      test-suite: "transfer"
```

Comment:
these strings map to sub directories, I'm open to alternatives if there are any suggestions!

Reply:
Will we also need to add blocks for `interchain-accounts` here?

Reply:
maybe we could rename the value to be something explicit like `matrix-filename` WDYT?

Reply:
I'm easy. I think its fine to leave as is

Reply:
Whatever is fine by me!